src/main/g8/src/main/scala/com/lightbend/akka/sample/HelloAkka.scala

```diff
@@ -0,0 +1,97 @@
+//#full-example
+package com.lightbend.akka.sample
+
+import akka.actor.{ ActorRef, ActorSystem, Props, Actor, ActorLogging }
+import scala.concurrent.duration._
+import scala.io.StdIn
+
+//#greeter-companion
+//#greeter-messages
+object Greeter {
+  //#greeter-messages
+  def props(message: String, printerActor: ActorRef): Props = Props(new Greeter(message, printerActor))
+  //#greeter-messages
+  case class WhoToGreet(who: String)
+  case object Greet
+}
+//#greeter-messages
+//#greeter-companion
+
+//#greeter-actor
+class Greeter(message: String, printerActor: ActorRef) extends Actor {
+  import Greeter._
+  import Printer._
+
+  var greeting = ""
+
+  def receive = {
+    case WhoToGreet(who) => greeting = s"$message, $who"
+    case Greet           =>
+      //#greeter-send-message
+      printerActor ! Greeting(greeting)
+      //#greeter-send-message
+  }
+}
+//#greeter-actor
+
+//#printer-companion
+//#printer-messages
+object Printer {
+  //#printer-messages
+  def props: Props = Props[Printer]
+  //#printer-messages
+  case class Greeting(greeting: String)
+}
+//#printer-messages
+//#printer-companion
```

Comment:
it's a bit unfortunate to have all these inclusion comments in the source code, when this is used a a "real" seed, and not as a tutorial. Wonder if we should inline/copy the code into the tutorial for this one, or can we have a copy of the source files that is only used by paradox but not included in the g8 download?